tests/sanity/tests/collaborative/issues.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test } from '@playwright/test'
+import { generateId, PlatformSetting, PlatformSettingSecond, PlatformURI } from '../utils'
+import { allure } from 'allure-playwright'
+import { NewIssue } from '../model/tracker/types'
+import { IssuesPage } from '../model/tracker/issues-page'
+import { LeftSideMenuPage } from '../model/left-side-menu-page'
+import { IssuesDetailsPage } from '../model/tracker/issues-details-page'
+
+test.use({
+  storageState: PlatformSetting
+})
+
+test.describe('Collaborative test for issue', () => {
+  test.beforeEach(async ({ page }) => {
+    await allure.parentSuite('Collaborative test')
+    await (await page.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+  })
+
+  test('Issues can be assigned to another users', async ({ page, browser }) => {
+    const newIssue: NewIssue = {
+      title: `Collaborative test for issue-${generateId()}`,
+      description: 'Collaborative test for issue',
+      status: 'Backlog',
+      priority: 'Urgent',
+      assignee: 'Appleseed John',
+      createLabel: true,
+      labels: `CREATE-ISSUE-${generateId()}`,
+      component: 'No component',
+      estimation: '2',
+      milestone: 'No Milestone',
+      duedate: 'today',
+      filePath: 'cat.jpeg'
+    }
+
+    await (await page.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+    const leftSideMenuPage = new LeftSideMenuPage(page)
+    await leftSideMenuPage.buttonTracker.click()
+
+    const issuesPage = new IssuesPage(page)
+    await issuesPage.createNewIssue(newIssue)
+    await issuesPage.linkSidebarAll.click()
+    await issuesPage.modelSelectorAll.click()
+    await issuesPage.searchIssueByName(newIssue.title)
+    await issuesPage.openIssueByName(newIssue.title)
+
+    // check by another user
+    const userSecondContext = await browser.newContext({ storageState: PlatformSettingSecond })
```

Comment:
I think you need to move it into some utils function in next PR.

Reply:
will do it, thank you =)

Reply:
done =)